### PR TITLE
Add more examples to ci

### PR DESCRIPTION
**Pull request by Luni-4**
Unfortunately the `ffmsinfo` executable can't read the audio track from the mkv file contained in the `ffms2/data` directory and this produces an error in the ci.

I've tried to run it locally using the latest `ffmpeg` version but the error remains the same.

I think it's the same problem highlighted by @lachs0r in #3

### CODE REVIEW

**Comment by Luni-4** [comment]
@rr- 

I decided to add a `C++` example to evaluate when an issue is caused by these bindings. In this case, it seems that the audio track of the video, used as example, can't be read by the `ffms2` library either.

I do think we should merge this PR and investigate this issue later, what do you think about it?

File: ffms2/data/ffmsinfo.cpp [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    printf("\tDuration = %f\n", VP->LastTime);
+    printf("\tNum Frames = %d\n\n", VP->NumFrames);
+
+    // Get Audio Track
+    audio_track_idx = FFMS_GetFirstTrackOfType(index, FFMS_TYPE_AUDIO, &E);
+    CHECK(audio_track_idx, 1);
+
+    audio_source = FFMS_CreateAudioSource(fileName.c_str(),
+                                          audio_track_idx, index,
+                                          FFMS_DELAY_FIRST_VIDEO_TRACK, &E);
```
Comment (rr-) [inline ffms2/data/ffmsinfo.cpp]: No against this kind of indentation

File: ffms2/data/ffmsinfo.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    FFMS_AudioSource* audio_source = nullptr;
+    const FFMS_AudioProperties* AP = nullptr;
+
+    FFMS_ErrorInfo E;
+    char ErrorMsg[1024];
+
+    E.Buffer = ErrorMsg;
+    E.BufferSize = sizeof(ErrorMsg);
+
+    FFMS_Init(0,0);
```
Comment (rr-) [inline ffms2/data/ffmsinfo.cpp]: Missing space

File: ffmsinfo.py [hunk 3]
```diff
@@ -93,7 +94,7 @@ def main():
 
             print("format =", format_name)
 
-            for n, (type_, codec_name) in enumerate(track_info_list):
+            for (n, type_, codec_name) in track_info_list:
```
Comment (rr-) [inline ffmsinfo.py]: Wait, why was this changed?

File: ffms2/data/ffmsinfo.cpp (hunk 1 above)
Comment (Luni-4) [reply ffms2/data/ffmsinfo.cpp]: Which one then?

File: ffmsinfo.py (hunk 3 above)
Comment (Luni-4) [reply ffmsinfo.py]: Because of this [line](https://github.com/bubblesub/pyffms2/blob/8a2214bc2747da6f491e97d33bd73216109fb506/ffms2/__init__.py#L336)

File: ffms2/data/ffmsinfo.cpp (hunk 1 above)
Comment (rr-) [reply ffms2/data/ffmsinfo.cpp]:
```
    audio_source = FFMS_CreateAudioSource(
        fileName.c_str(),
        audio_track_idx,
        index,
        FFMS_DELAY_FIRST_VIDEO_TRACK,
        &E
    );

**Comment by rr-** [comment]
LMK when it's ready to get merged

**Comment by Luni-4** [comment]
> LMK when it's ready to get merged

For me it's ready, but it depends on what you want to do about the CI failure.

**Comment by rr-** [comment]
Would it be possible for you to fix it in a separate pr?

**Comment by Luni-4** [comment]
Yeah, I'm going to change the video example, it's not a fix, but at least the CI works. 

Then I could investigate why `ffms2` can't read the audio track of the current video example.

So feel free to land this PR :)

File: ffms2/data/ffmsinfo.cpp [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        &E
+    );
+
+    NULL_CHECK(video_source);
+
+    VP = FFMS_GetVideoProperties(video_source);
+
+    printf("\n\nVideo properties\n\n");
+    printf("\tFPS = %f\n",
+           (VP->FPSNumerator / static_cast<double>(VP->FPSDenominator)));
```
Comment (rr-) [inline ffms2/data/ffmsinfo.cpp]: Still old indentation

**Comment by Luni-4** [comment]
BTW the problem happens with other videos too. I'm going to bisect `ffms2` in order to find the problem, but I think it's located in the `indexer.cpp` file.

**Comment by rr-** [comment]
💙
